src/stream/src/executor/sync_kv_log_store.rs

```diff
@@ -404,12 +487,13 @@ impl<S: StateStore> SyncedKvLogStoreExecutor<S> {
                                     epoch,
                                 );
                                 flush_info.report(&self.metrics);
-                                write_future =
+                                write_future_state =
                                     WriteFuture::receive_from_upstream(stream, write_state);
                             }
                         }
                     }
                     Either::Right(result) => {
+                        println!("read_stream progress");
```

Comment:
It seems that we need to update `clean_state  = true` here when some conditions are met. In current code, we have no where to reset `clean_state  = true`.

Reply:
Fixed, reset clean state, if read future in an idle state, and if the buffer has no flushed items.